### DISCUSSION TITLE
How I can set a default date using the singleCalendar option?

### DISCUSSION THREAD

**Discussion by clizaola** [malzariey/filament-daterangepicker-filter#142] [Q&A]
Hi, I am unsure if this is the best way, but I am trying to set the default date to open the calendar on the day, month, and year set on the default using the singleCalendar option. This is to be able to edit the Date of Birth.

DateRangePicker::make('dob')
                ->default('1990-01-01')
                ->singleCalendar()
                ->showDropdowns()
                ->disableRanges()
                ->autoApply()
                ->format('DD/MM/YYYY')
                ->displayFormat('DD/MM/YYYY'),

**Comment by malzariey** [comment]
Thank you for using the plugin:

You would need to specify the dates with the seperator and match the format in the ->default function

or using the ->startDate() and ->endDate() functions

https://github.com/malzariey/filament-daterangepicker-filter?tab=readme-ov-file#start-and-end-dates
